src/less/filter-menu-button/base/filter-menu-button.less

```diff
@@ -0,0 +1,247 @@
+        height: 7px;
+        width: 12px;
+    }
+
+    & > span.icon {
+        .background-icon-base-mixin();
+        .background-icon-image-mixin('chevron-down', 12px, 7px);
+    }
+
+    svg.icon,
+    span.icon {
+        align-self: center;
+        margin-left: 8px;
+    }
+}
+
+.filter-menu-button__button-text {
+    display: inline-block;
+    max-width: 238px;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+}
+
+.filter-menu-button__button[aria-expanded="true"] .filter-menu-button__button-cell > .icon {
+    transform: rotate(180deg);
+}
+
+button.filter-menu-button__button[aria-pressed="true"] {
+    .filter-button-selected();
+
+    background-color: @filter-button-selected-background-color; // should we just lighten @color-b4;
+    color: @filter-button-selected-color;
+
+    &:focus,
+    &:hover,
+    &:active {
+        background-color: @filter-button-selected-hover-background-color; // should we just lighten @color-b4;
+    }
+}
+
+button.filter-menu-button__button[aria-expanded="true"][aria-pressed="true"] {
+    background-color: @filter-button-selected-hover-background-color; // should we just lighten @color-b4;
+}
+
+button.filter-menu-button__button[disabled],
+button.filter-menu-button__button[aria-disabled="true"],
+button.filter-menu-button__button[aria-pressed="true"][disabled],
+button.filter-menu-button__button[aria-pressed="true"][aria-disabled="true"] {
+    .filter-button-disabled();
+
+    color: @filter-button-disabled-color;
+
+    &:hover,
+    &:focus {
+        background-color: @filter-button-background-color;
+        color: @filter-button-disabled-color;
+    }
+
+    & .filter-menu-button__button-cell > svg.icon {
+        color: @filter-button-disabled-color;
+    }
+
+    & .filter-menu-button__button-cell > span.icon {
+        .background-icon-image-mixin('chevron-down-disabled', 12px, 7px);
+    }
+}
+
+button.filter-menu-button__button[aria-pressed="true"][disabled]:hover,
+button.filter-menu-button__button[aria-pressed="true"][aria-disabled="true"]:hover {
+    background-color: @filter-button-selected-background-color;
+}
+
+.filter-menu-button__menu {
+    background-color: @filter-menu-item-background-color;
+    border: none;
+    border-radius: 16px;
+    box-shadow: 0 0 5px 0 rgba(0, 0, 0, 0.5);
+    display: none;
+    max-width: 288px; // TODO remove max-width restriction to filter-menu styles
+    min-width: 144px;
+    overflow: hidden;
+    position: absolute;
+    top: calc(100% + 8px);
+    width: max-content;
+    z-index: 1;
+}
+
+button.filter-menu-button__button[aria-expanded="true"] + .filter-menu-button__menu {
+    display: block;
+}
+
+.filter-menu-button__items {
+    margin-top: 8px;
+    max-height: calc(50vh - 40px); // half of the viewport height || TODO remove max-height restriction to filter-menu styles
+    min-width: 100%;
+    overflow-y: auto;
+    position: relative;
+}
+
+span.filter-menu-button__items {
```

Comment:
Could we change the __items to something else, like menu or something. It was really confusing trying to discern between __item and __items. I know we did the same with listbox and such and it's much cleaner.
I'm fine with doing it as a refactor if it's too complex.

Reply:
@ianmcburnie This followed your `menu` pattern. Should we consider changing it in both places.?

Reply:
Let's leave as is for now. I'll have a think.